modules/addListForm.ts

```diff
@@ -0,0 +1,48 @@
+import {
```

Comment:
On the same note as the modal comment - we're doing a lot of similar stuff in the forms. We could plan a way to share between them more easily.